src/libs/auth.ts

```diff
@@ -126,10 +127,21 @@ export const signOut = (cause: SignOutCause = 'unspecified'): void => {
   azureCookieReadyStore.reset();
   getSessionStorage().clear();
   azurePreviewStore.set(false);
+
   const auth: AuthContextProps = getAuthInstance();
   revokeTokens()
     .finally(() => auth.removeUser())
     .finally(() => auth.clearStaleState());
+  const cookiesAccepted: boolean | undefined = authStore.get().cookiesAccepted;
+  authStore.reset();
```

Comment:
Do we need to clear `user` out of `oidcStore` here as well?

Reply:
Yep, missed that 👍